www/index.html

```diff
@@ -5,49 +5,46 @@
     <meta name="viewport" content="initial-scale=1, maximum-scale=1, user-scalable=no, width=device-width">
     <title>HI Mobility</title>
 
-    <!-- compiled css output -->
-    <link href="css/ionic.app.css" rel="stylesheet">
-    <link href="http://netdna.bootstrapcdn.com/font-awesome/4.0.3/css/font-awesome.css" rel="stylesheet">
+    <!-- Vendor CSS -->
+    <link rel="stylesheet" href="bower_components/fontawesome/css/font-awesome.min.css">
     <link rel="stylesheet" href="css/leaflet.css" />
+
+    <!-- Application CSS -->
+    <link rel="stylesheet" href="css/ionic.app.css">
     <link rel="stylesheet" href="css/style.css" />
 
-    <!-- Angulars Material CSS -->
+    <!-- Angular Material CSS -->
+    <!-- This must be referenced after css/style.css -->
     <link rel="stylesheet" href="bower_components/angular-material/angular-material.min.css">
 
-    <!-- ionic/angularjs js -->
+    <!-- Ionic JS -->
```

Comment:
Not a big deal, but there's a lot of script and style files. If there's time, it might be good to start looking into concatenation + minification.